static/src/js/components/CollapsePanel/CollapsePanel.jsx

```diff
@@ -49,6 +49,7 @@ export class CollapsePanel extends Component {
         <button
           className={buttonClassNames}
           type="button"
+          aria-label="collapse-text"
```

Comment:
This label should be human readable and indicate clearly what the action will do. "collapse-text" doesnt make much sense.

Reply:
So two things:
- By human readable, I meant something _not_ kebab cased.
- This label is for accessibility. When a user users assistive technologies, this label is what will be read back to them. Given that, it should be very clear what it does. We should make it dynamic so that when the panel is closed, it says "Open panel" and when it is open, it says "Close panel." 

Even better, if it didn't add too much creep, we could add a prop that allowed us to pass an optional name, so that if set, we could dynamically display text that said "Open Advanced Options panel"/"Close Advanced Options panel" which would likely be more helpful to users. I could also see that being a separate improvement ticket.

Reply:
Added and included a test rewrite for the collapsable panel

Reply:
Looking back I really like that previous idea you had. This is actually only used in two places right now but, I see the semantic value to what you are saying. I'm going to work that into the next ticket I do as we have the swodlr enhancements epic this next PI